app/src/main/java/at/aau/serg/activities/GameScreenActivity.kt

```diff
@@ -155,11 +156,13 @@ class GameScreenActivity : AppCompatActivity() {
         val suit = (socketResponse[0] as JSONObject).getString("suit")
         val value = (socketResponse[0] as JSONObject).getString("value")
         val playerIdx = (socketResponse[0] as JSONObject).getInt("playerIdx")
+        val winnerIdx = (socketResponse[0] as JSONObject).getInt("winnerIdx")
```

Comment:
Lets avoid repeating the ` socketResponse[0] as JSONObject` cast four times

Reply:
solved